src/test/java/selenium/test/CreateCandidatePageTest.java

```diff
@@ -0,0 +1,35 @@
+package selenium.test;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import selenium.model.Candidate;
+import selenium.page.CreateCandidatePage;
+
+public class CreateCandidatePageTest extends BaseTest {
+    @Test
+    public void createCandidateTest() {
+        Candidate candidate = new Candidate();
+        CreateCandidatePage page = getLoginPage()
+                .fillUsername(getCredentials().getUserLoginName())
+                .fillPassword(getCredentials().getUserPassword())
+                .clickLoginButton()
+                .moveToRecruitmentItem()
+                .clickCandidateListButton()
+                .clickNewCandidateButton()
+                .fillFirstName(candidate.getCandidateFirstName())
+                .fillLastName(candidate.getCandidateLastName())
+                .fillEmail(candidate.getCandidateEmail())
+                .fillContactNo(candidate.getCandidateContactNo())
+                .selectVacancy(candidate.getCandidateVacancy())
+                .fillDate(candidate.getCandidateAppliedDate())
+                .btnSaveClick();
+
+
+        if (page.checkValidation()) {
+            String successMessage = page.getSuccessMessage();
+            Assert.assertEquals(successMessage, "Successfully Saved\nClose");
+        } else {
+            Assert.fail("Check required fields! Fields message: \n" + page.getValidationMessage());
+        }
```

Comment:
Please avoid if/else in assertion part, expected result should be only one

Reply:
Fixed